charts/hawkbit-update-server/values.yaml

```diff
@@ -104,8 +116,19 @@ config:
       datasource:
         username: hawkbit
         password: hawkbit
-  application:
-    spring:
-      security:
-        user:
-          name: admin
+
+
+# dependency charts config
+mysql:
+  mysqlUser: hawkbit
+  mysqlPassword: hawkbit
+  mysqlDatabase: hawkbit
+  metrics:
+    enabled: true
+
+rabbitmq:
```

Comment:
instead of the fields `useRabbitmq` I think would be more consistent with a `rabbitmq.enabled` field.
What do you think?